SPECS/sqlite/sqlite.spec

```diff
@@ -1,19 +1,20 @@
-%define sourcever 3320300
+%define sourcever 3340100
 Summary:        A portable, high level programming interface to various calling conventions
 Name:           sqlite
-Version:        3.32.3
-Release:        2%{?dist}
+Version:        3.34.1
+Release:        1%{?dist}
 License:        Public Domain
-URL:            https://www.sqlite.org
-Group:          System Environment/GeneralLibraries
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:        https://www.sqlite.org/2020/%{name}-autoconf-%{sourcever}.tar.gz
-# CVE-2015-3717 applies to versions shipped in iOS and OS X 
+Group:          System Environment/GeneralLibraries
```

Comment:
Since groups are obsolete and bring no value, I suggest removing this bit.

Reply:
Going to go ahead and leave it for now- guidance last time I checked (around October) was that this may eventually be something we want to use.